TramsDataApi/UseCases/GetAllFssProject.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using TramsDataApi.Factories;
+using TramsDataApi.Gateways;
+using TramsDataApi.ResponseModels;
+
+namespace TramsDataApi.UseCases
+{
+    public class GetAllFssProject : IGetAllFssProject
```

Comment:
Can this be `GetAllFssProjects : IGetAllFssProjects`?

Reply:
Yes can be